docs/source/models/gbd2017/causes/ischemic_heart_disease/index.rst

```diff
@@ -11,37 +11,65 @@ Ischemic heart disease (IHD) is a non-communicable cardiovascular disease which
 
 Acute coronary events may cause symptoms such as angina, cold sweats, dizziness, nausuea, neckpain, shortness of breath, sleep disturbances, or weakness. 
 
-Chronic ischemic heart disease can cause signs and symptoms such as angina, anxiety or nervousness, fatigue, or neckpace. 
+Chronic ischemic heart disease can cause signs and symptoms such as angina, anxiety or nervousness, fatigue, or neckpain. 
 
 .. _NIH: https://www.nhlbi.nih.gov/health-topics/ischemic-heart-disease
 
 
-Modeling Ischemic Heart Disease in GBD 2017
--------------------------------------------
+GBD 2017 Modeling Strategy
+-------------------------------------------------
+
+GBD 2017 models fatal and non-fatal IHD estimates separately. GBD 2017 models IHD using Myocardial infarction (MI) sequelae to estimate the prevalence of IHD, due to the challenges of disease detection and varying symptoms across the population.
 
 Myocardial infarction (MI) in GBD 2017
 ++++++++++++++++++++++++++++++++++++++
 
+The case definition can be found below and is stratified by two sections: 'Acute myocardial infarction (MI)' and 'Chronic IHD'. The case definition for non-fatal IHD modeling can be found in both sections of the case definition. The case definition for fatal IHD modeling can be found in section 1 below.
 
-Modeling Strategy for Ischemic Heart Disease
-++++++++++++++++++++++++++++++++++++++++++++
+1. Acute myocardial infarction (MI): Definite and possible MI according to the third universal definition of myocardial infarction:
 
-Cause Model Diagram
---------------------
-.. image:: ischemic_heart_disease_transitions.svg
-  :width: 150
+When there is clinical evidence of myocardial necrosis in a clinical setting consistent with
```

Comment:
Not super important, but it might be nice to indent (or otherwise visually distinguish) everything that is a subset of "1.", so that it's clear what falls under that definition

Reply:
Great point. Just updated that!